dashboard/app/mailers/pd/application/teacher_application_mailer.rb

```diff
@@ -95,6 +95,24 @@ def registration_reminder(teacher_application)
       )
     end
 
+    def complete_application_soft_reminder(teacher_application)
+      @application = teacher_application
```

Comment:
Nit: Would it be possible to change the naming of this from "soft" to "first" or something like that? I think reading the emails here in this file it's easy to tell that these two emails are connected and that the only difference between them is timing but I think these names in other files would not intimate that timing is what distinguishes them. Super minor nit though, so probably not worth a rebuild if there are no other updates, just a thought!

Reply:
I originally had these as "first" and "second" reminder but there's no logic that actually ensures that the first is received before the second.

I was imagining a weird scenario where our mailer system is down and we miss the first ("soft") email and will instead only send the second ("hard") one. But perhaps that's a bit too far-fetched to necessitate more confusing wording.

Reply:
Oh I see, it's a tiny thing so no big deal either way, totally your call!